client/ayon_core/hosts/houdini/api/lib.py

```diff
@@ -244,13 +245,14 @@ def render_rop(ropnode):
         ropnode.render(verbose=verbose,
                        # Allow Deadline to capture completion percentage
                        output_progress=verbose)
-    except hou.Error as exc:
+    except hou.Error:
         # The hou.Error is not inherited from a Python Exception class,
         # so we explicitly capture the houdini error, otherwise pyblish
         # will remain hanging.
         import traceback
         traceback.print_exc()
-        raise RuntimeError("Render failed: {0}".format(exc))
+        raise KnownPublishError("'{}' failed to render!".format(ropnode.path()),
+                                label="Render was interrupted or canceled.")
```

Comment:
This isn't 100% true. It _may_ have been canceled or interrupted - but it may also have been an actual error during the ROP render. As such, the message here may be untrue.

Also, what's the difference between canceled and interrupted? Isn't that essentially the same thing here?